static/includes/CreateCloudSyncTaskScale.md

```diff
@@ -1,33 +1,34 @@
 &NewLine;
 
-To add a cloud sync task, go to **Data Protection > Cloud Sync Tasks** and click **Add**. The **Cloudsync Task Wizard** opens.
+To add a cloud sync task, go to **Data Protection > Cloud Sync Tasks** and click **Add**. The **Cloud Sync Task Wizard** opens.
 
-![CloudSyncTaskWizardProviderScreen](/images/SCALE/DataProtection/CloudSyncTaskWizardProviderScreen.png "Cloudsync Task Wizard Provider")
+{{< trueimage src="/images/SCALE/DataProtection/CloudSyncTaskWizardProviderScreen.png" alt="Cloud Sync Task Wizard" id="Cloud Sync Task Wizard" >}}
 
 1. Select an existing backup credential from the **Credential** dropdown list.
+   
    If not already added as a cloud credential, click **Add New** to open the **Cloud Credentials** screen to add the credential.
    Click **Save** to close the screen and return to the wizard.
 
-2. Click **Next** to open the **Where and When** wizard screen.
+   Click **Verify Credential** to ensure the credentials are valid then click **Next**.
 
-   ![CloudSyncTaskWizardWhatandWhenScreen](/images/SCALE/DataProtection/CloudSyncTaskWizardWhatandWhenScreen.png "Cloudsync Task Wizard What and When")
-
-3. Select the option from **Direction** and in **Transfer Mode**.
+2. Select the option from **Direction** and in **Transfer Mode**.
    Select the location where to pull from or push data to in the **Folder** field.
 
-4. Select the dataset location in **Directory/Files**. Browse to the dataset to use on SCALE for data storage.
+   {{< trueimage src="/images/SCALE/DataProtection/CloudSyncTaskWizardWhatandWhenScreen.png" alt="Cloud Sync Task Wizard What and When" id="Cloud Sync Task Wizard - What and When" >}}
+
+3. Select the dataset location in **Directory/Files**. Browse to the dataset to use on SCALE for data storage.
    Click the arrow to the left of the name to expand it, then click on the name to select it.
 
    If **Direction** is set to **PUSH**, click on the folder icon to add **/** to the **Folder** field.
 
    {{< include file="/static/includes/FileExplorerFolderIcons.md" >}}
 
-5. Cloud provider settings change based on the credential you select. Select or enter the required settings that include where files are stored.
+4. Cloud provider settings change based on the credential you select. Select or enter the required settings that include where files are stored.
```

Comment:
For step procedures, it is better to make the action the first thing in this step then the clarification. Also it is better parallel construction of the procedure. One suggestion for changing this step:
```suggestion
4. Select or enter the required settings that include where files are stored. Cloud provider settings change based on the credential selected. 
```